src/store/modules/models/libconfig.js

```diff
@@ -58,6 +58,45 @@ const map = {
     ],
     init: factory.ThermalZone,
   },
+  window_definitions: {
```

Comment:
Are window and daylighting control instances meant to be included in this file? we may want to remove those unless they are being used somewhere

Reply:
I don't think I understand. Where else would they be? Isn't this how we define something that will live in the library?